sru-scripts/manual/oracle-sru

```diff
@@ -17,13 +18,34 @@ assert_expected_vars PROPOSED_SCRIPT SAMPLE_CLOUDCONFIG LP_USER SSHOPTS \
 create_samplecloudconfig SAMPLE_CLOUDCONFIG
 create_setup_proposed_script
 
+function cleanup {
+    echo "Cleaning up instance ${INSTANCE_ID}"
+    oci compute instance terminate --force --instance-id="${INSTANCE_ID}"
+}
 
 echo "### BEGIN $UBUNTU_RELEASE"
 
-# TODO: Use the Oracle CLI
-echo "Please manually launch a $UBUNTU_RELEASE instance in Oracle and enter its IP address"
-echo "(Don't forget to add your SSH public key to the instance when you launch it.)"
-read -p "$UBUNTU_RELEASE IP address: " VM_IP
+declare -A releases
+releases=( ["xenial"]="16.04" ["bionic"]="18.04" ["focal"]="20.04" )
+
+IMAGE_ID=$(oci compute image list --operating-system="Canonical Ubuntu" --operating-system-version="${releases[${UBUNTU_RELEASE}]}" --sort-by="TIMECREATED" --limit=1 2>/dev/null | jq -r '.data[0]["id"]')
```

Comment:
Since this command fails to produce results if you don't have a default compartment-id set. We should add a test a suggestion for the fix

if [ -z "${IMAGE_ID}" ]; then
  echo "Unable to list images, ensure default compartment-id is set in ~/.oci/oci_cli_rc
  exit 1
fi